api/pictures.py

```diff
@@ -0,0 +1,188 @@
+from flask import request as req, jsonify, send_file
+import os
+
+import lib.file_upload as file_upload
+import lib.exceptions as exceptions
```

Comment:
O que acha dessa outra abordagem mais pythonica?
```suggestion
from lib import file_upload, exceptions
```

Reply:
Perfeito, Rocky! Eu conhecia essa sintaxe para módulos nativos do Python, mas não tinha me dado conta de que poderia usá-la para meus próprios módulos